pydm/widgets/qtplugins.py

```diff
@@ -266,3 +261,7 @@
 PyDMTerminatorPlugin = qtplugin_factory(PyDMTerminator,
                                         group=WidgetCategory.MISC,
                                         extensions=BASE_EXTENSIONS)
+
+
+# Add in designer widget plugins from other classes via entrypoints:
+globals().update(**get_widgets_from_entrypoints())
```

Comment:
Using `globals().update()` implies that we may want a follow-up to change how these are discovered- like importing a function to return all the widgets, instead of star importing from this module. An annoying change of course, out of scope here.

Reply:
This could be "todo" issue-worthy. The end result may still be an `from pydm.widgets.qtplugins import *` due to how pyqt designer extensions work, but there may be a cleaner way to create and aggregate these.